### PR TITLE
Workaround delayed log file creation

### PR DESCRIPTION
**Pull request by apostasie**
Split out of PR #3178 

The issue is documented in #3187 

This is not a proper fix - it is a (hopefully temporary) workaround so that we stop failing `TestTailFollowRotateLogs` so often.

### CODE REVIEW

**Comment by apostasie** [comment]
cc @AkihiroSuda

File: pkg/logging/json_logger.go [hunk 1]
```diff
@@ -129,7 +130,15 @@ func (jsonLogger *JSONLogger) PostProcess() error {
 func viewLogsJSONFile(lvopts LogViewOptions, stdout, stderr io.Writer, stopChannel chan os.Signal) error {
 	logFilePath := jsonfile.Path(lvopts.DatastoreRootPath, lvopts.Namespace, lvopts.ContainerID)
 	if _, err := os.Stat(logFilePath); err != nil {
-		return fmt.Errorf("failed to stat JSON log file ")
+		// FIXME: this is a workaround for the actual issue, not a real solution
```
Comment (AkihiroSuda) [inline pkg/logging/json_logger.go:133]: Could you add a link to the github issue?

Comment (apostasie) [reply pkg/logging/json_logger.go:133]: Done

**Comment by apostasie** [comment]
Comment addressed.
PTAL at your convenience @AkihiroSuda

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**